### PR TITLE
l10n-related documentation improvements

### PR DESCRIPTION
**Pull request by Hixie**

### CODE REVIEW

File: dev/tools/localization/bin/gen_localizations.dart [hunk 1]
```diff
@@ -2,11 +2,11 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// This program generates a getMaterialTranslation() and a
-// getCupertinoTranslation() function that look up the translations provided by
+// This program generates getMaterialTranslation(), getCupertinoTranslation(),
+// and getWidgetsTranslation functions that look up the translations provided by
```
Comment (goderbauer) [inline dev/tools/localization/bin/gen_localizations.dart:6]: nit: stick to one style, either use `getFooTranslation` or `getFooTranslation()` everywhere in this enumeration?

File: dev/tools/localization/bin/gen_localizations.dart [hunk 2]
```diff
@@ -543,19 +543,19 @@ void main(List<String> rawArgs) {
...
   final Map<LocaleInfo, Map<String, String>> materialLocaleToResources = <LocaleInfo, Map<String, String>>{};
-  // Maps of locales to resource key/attributes pairs for Material ARBs..
+  // Maps of locales to resource key/attributes pairs for Material ARBs.
   // https://github.com/googlei18n/app-resource-bundle/wiki/ApplicationResourceBundleSpecification#resource-attributes
   final Map<LocaleInfo, Map<String, dynamic>> materialLocaleToResourceAttributes = <LocaleInfo, Map<String, dynamic>>{};
 
   // Maps of locales to resource key/value pairs for Cupertino ARBs.
   final Map<LocaleInfo, Map<String, String>> cupertinoLocaleToResources = <LocaleInfo, Map<String, String>>{};
-  // Maps of locales to resource key/attributes pairs for Cupertino ARBs..
+  // Maps of locales to resource key/attributes pairs for Cupertino ARB.
```
Comment (goderbauer) [inline dev/tools/localization/bin/gen_localizations.dart:558]: Is the change from plural ARBs to singular ARB intentional? Is Cupertino special that way compared to Material and Widgets above?

**Review by goderbauer — APPROVED** [review]
LGTM

File: dev/tools/localization/bin/gen_localizations.dart (hunk 1 above)
Comment (Hixie) [reply dev/tools/localization/bin/gen_localizations.dart:6]: added ()

File: dev/tools/localization/bin/gen_localizations.dart (hunk 2 above)
Comment (Hixie) [reply dev/tools/localization/bin/gen_localizations.dart:558]: no that was just a typo, fixed

**Comment by Hixie** [comment]
thanks for the detailed and careful review, much appreciated!

**Pull request merged**